Patient Registry System/PatientRegistrySystem.API/Controllers/UsersController.cs

```diff
@@ -12,38 +12,48 @@ public class UsersController : Controller
     {
         public UsersController(IUserService userService)
         {
-            _userService = userService ?? throw new ArgumentNullException(nameof(userService));
+            UserService = userService;
         }
+        private readonly IUserService UserService;
```

Comment:
private props usually are in camelCase and starts with _
_userService

this applies to any private prop you have in the project